### ISSUE TITLE
False positive E1003 on Python 3 argument-less super()

### ISSUE THREAD

**Issue by pylint-bot** [PyCQA/pylint#16]
Originally reported by: **BitBucket: [Jajcus](http://bitbucket.org/Jajcus), GitHub: @Jajcus?**

---

Test Python 3 code (test.py):

```
:::python
class Base(object):
    def __init__(self):
        super().__init__()
```

Running 'pylint --include-ids=y --reports=n --disable=C0111,R0903 test'  (using Python3 pylint) gives:

```
E1003:  2,4:Base.__init__: Bad first argument %r given to super class
```

There are three problems with that:
- **it is a valid Python 3 usage of super()** (much more convenient that the one in Python 2), so **no error should be reported**
- '%r' is not substituted with anything
- super() is in line 3, not line 2

---
- Bitbucket: https://bitbucket.org/logilab/pylint/issue/16
